plugins/plugin-docker/che-plugin-docker-machine/src/main/java/org/eclipse/che/plugin/docker/machine/ext/DockerExtServerModule.java

```diff
@@ -50,6 +50,12 @@ protected void configure() {
                                      + org.eclipse.che.plugin.docker.machine.ext.provider.DockerExtConfBindingProvider
                                              .EXT_CHE_LOCAL_CONF_DIR);
 
+        Multibinder<String> allMachinesEnvVars = Multibinder.newSetBinder(binder(),
```

Comment:
It is wrong place to add these bindings. This module should be used to add things specific to ws agent